plugins/dmxusb/src/dmxusb.cpp

```diff
@@ -155,10 +155,10 @@ QString DMXUSB::outputInfo(quint32 output)
         if (m_outputs.size() == 0)
         {
             str += QString("<BR><B>%1</B>").arg(tr("No output support available."));
-            str += QString("<P>");
+            /*str += QString("<P>");
```

Comment:
Why?
Please uncomment.

Reply:
The purpose of the comment was to not mislead the user with this message, since this message will always be displayed even if all hardawre is correctly plugged.

I'll remove the comment, but perhaps a reword of this notice will be needed.